api/src/main/java/org/openmrs/module/eptsreports/reporting/calculation/generic/AgeInMonthsOnArtStartDateCalculation.java

```diff
@@ -0,0 +1,78 @@
+package org.openmrs.module.eptsreports.reporting.calculation.generic;
+
+import java.util.Collection;
+import java.util.Date;
+import java.util.Map;
+import org.joda.time.Interval;
+import org.joda.time.Months;
+import org.openmrs.api.context.Context;
+import org.openmrs.calculation.patient.PatientCalculationContext;
+import org.openmrs.calculation.result.CalculationResult;
+import org.openmrs.calculation.result.CalculationResultMap;
+import org.openmrs.module.eptsreports.reporting.calculation.AbstractPatientCalculation;
+import org.openmrs.module.eptsreports.reporting.calculation.BooleanResult;
+import org.openmrs.module.eptsreports.reporting.utils.EptsCalculationUtils;
+import org.openmrs.module.reporting.common.Birthdate;
+import org.openmrs.module.reporting.data.person.definition.BirthdateDataDefinition;
+import org.springframework.stereotype.Component;
+
+@Component
+public class AgeInMonthsOnArtStartDateCalculation extends AbstractPatientCalculation {
+
+  private static final String MAX_AGE = "maxAgeInMonths";
+
+  private static final String MIN_AGE = "minAgeInMonths";
+
+  @Override
+  public CalculationResultMap evaluate(
+      Collection<Integer> cohort,
+      Map<String, Object> parameterValues,
+      PatientCalculationContext context) {
+    CalculationResultMap map = new CalculationResultMap();
+
+    CalculationResultMap artStartDates =
+        calculate(
+            Context.getRegisteredComponents(InitialArtStartDateCalculation.class).get(0),
```

Comment:
Hi @ningosi this is not the right ArtStartDate, please use this new calculation 
```suggestion
            Context.getRegisteredComponents(MohMQInitiatedARTDuringTheInclusionPeriodCalculation.class).get(0),
```

Look at developer, that calculation is already there

Reply:
Ok, noted. I will use as advised